.github/workflows/dependency-review.yml

```
@@ -9,11 +9,6 @@
 name: 'Dependency Review'
 on: 
   pull_request:
-  merge_group:
-    paths-ignore:
-    - "LICENSE"
-    - "CODEOWNERS"
-    - "**.md"
```

```suggestion
    paths-ignore:
    - "LICENSE"
    - "CODEOWNERS"
    - "**.md"
```

PRs should still ignore these files, I think.

---

✅